precompiles/avs/IAVSManager.sol

```diff
@@ -49,6 +49,7 @@ interface IAVSManager {
     /// @param slashAddr The slash address of AVS.
```

Comment:
Have you verified that this file compiles? Too many local variables (within a function) can cause Solidity to complain "stack too deep"

Reply:
testing